react/features/filmstrip/functions.web.js

```diff
@@ -513,14 +513,23 @@ export function computeDisplayModeFromInput(input: Object) {
         isScreenSharing,
         canPlayEventReceived,
         isRemoteParticipant,
+        multipleVideoSupport,
         stageParticipantsVisible,
         stageFilmstrip,
         tileViewActive
     } = input;
     const adjustedIsVideoPlayable = input.isVideoPlayable && (!isRemoteParticipant || canPlayEventReceived);
 
-    if (isFakeScreenShareParticipant) {
-        return DISPLAY_VIDEO;
+    if (multipleVideoSupport) {
+        // Display video for fake screen share participants in all layouts.
+        if (isFakeScreenShareParticipant) {
+            return DISPLAY_VIDEO;
+        }
+
+        // For Plan B enbdpoints with multi-stream support, display an avatar for the original thumbnail.
+        if (isScreenSharing && isRemoteParticipant) {
```

Comment:
I'm not sure if we even need this change. Will there ever be a scenario where web on Plan B? What is it currently showing for the original tile in RN?